t/sysopen.t

```diff
@@ -0,0 +1,50 @@
+#! perl
+
+use strict;
+use warnings;
+
+use Test::More;
+use Errno 'ENOENT';
+
+use FindBin;
+use File::Temp;
+use lib "$FindBin::Bin/lib";
+use AutocroakTestUtils;
+
+use Fcntl;
+
+subtest success => sub {
+    use autocroak;
+
+    my ($tfh, $tpath) = File::Temp::tempfile( CLEANUP => 1 );
+
+    sysopen my $fh, $tpath, Fcntl::O_RDONLY;
+
+    my @stat1 = stat $tfh;
+    my @stat2 = stat $fh;
+
+    is( "@stat1", "@stat2" );
+};
+
+subtest basic => sub {
+	use autocroak;
+    my $enoent = AutocroakTestUtils::get_errno_string('ENOENT');
```

Comment:
Accidental spaces?

Reply:
Oops. Can fix later.

Reply:
Actually not accidental, just inconsistent use of tabs vs. spaces owing to copy/paste from your existing test. Fixed.